draft-polli-ratelimit-headers.md

```diff
@@ -1027,3 +1034,23 @@ RateLimit-Limit: 100, 100;w=60;burst=1000;comment="sliding window", 5000;w=3600;
 
    the key value is the one referencing the lowest limit: `100`
 
+10. How can I define the throttling scope? To which target does the `RateLimit-Remaining` apply?
+
+   This is a very good question. Actually we should answer this question for the `Retry-After` header first :)
+   Once we have a meaningful answer for that standard header, we can adapt the solution to the `RateLimit` ones.
+
+   To overcome this situation we specified that the provided values were the one used to evaluate
+   the current request. See {{providing-ratelimit-headers}}.
+
+   There are many possible solutions, included the ones below. The point is just to find one which is
+   consistent with the rest of the HTTP spec.
+
+   - pass the information in a different header (eg `Warning`, `RateLimit-Scope`)
+   - add a custom parameter to `RateLimit-Limit` (this spec already allows that, like shown
+     in the following two examples
```

Comment:
Perhaps hint at a future I-D extending this one? From the comment I've written above there is also the question of how to return information about more than one resource at once because both might be taken into account when servicing a request, such as connections and specific service endpoints (and these two in particular would also be specified in different ways, as you can't refer to connections via a path). We could also defer discussion to such an I-D.

Reply:
I think we should:

- gather clear requirements (eg. request/response) for those use cases;
- try to address them one by one;
- find a common pattern, if exists or delegate to another I-D.

I will clarify that we *will* face the issue in the next releases, but not now as we're starting up.
I'll modify the slides accordingly.